src/components/header.js

```diff
@@ -13,23 +16,23 @@ const AppTitle = styled.h1`
   flex-grow: 1;
 `
 
-const Header = ({ darkMode, isLoggedIn, logout, toggleDarkMode }) => {
+const Header = () => {
+  const darkMode = useSelector(getDarkMode)
+  const loggedIn = useSelector(isLoggedIn)
+
+  const dispatch = useDispatch()
+  const handleLogout = () => dispatch(logout())
+  const handleToggleDarkMode = () => dispatch(toggleDarkMode())
```

Comment:
I think with the release of https://react-hooks-testing-library.com/usage/basic-hooks it's easy to test hooks, which in turn makes it easy to introduce 
```js
import { useDarkModeToggle, useUserLoginStatus } from '../store/hooks'
```
and then just
```suggestion
  const [darkMode, toggleDarkMode] = useDarkModeToggle()
  const [isLoggedIn, logout] = useUserLoginStatus()
```

Reply:
Yes, that could be another approach. I'm not excited with the idea of having to export every selector/action wrapped in useDispatch or useSelector, just to simplify testing a bit. Although definitely it's better that this exploration.

Regarding `react-hooks-testing-library`, it allows us to standalone-test the hook functionality, not the component behaviour when this hook changes 😕